modules/installation-overview.adoc

```diff
@@ -7,27 +7,27 @@
+* *Local Agent-based*: You can deploy a cluster locally with the Agent-based Installer for disconnected environments or restricted networks. It provides many of the benefits of the {ai-full}, but you must download and configure the link:https://console.redhat.com/openshift/install/metal/agent-based[Agent-based Installer] first. Configuration is done with a command-line interface. This approach is ideal for disconnected environments or restricted networks.
 
-* *Automated*: You can deploy a cluster on installer-provisioned infrastructure and the cluster it maintains. The installer uses each cluster host's baseboard management controller (BMC) for provisioning. You can deploy clusters with both connected or air-gapped or restricted networks.
+* *Automated*: You can deploy a cluster on installer-provisioned infrastructure. The installation program uses each cluster host's baseboard management controller (BMC) for provisioning. You can deploy clusters in connected or disconnected environments.
 
-* *Full control*: You can deploy a cluster on infrastructure that you prepare and maintain, which provides maximum customizability. You can deploy clusters with both connected or air-gapped or restricted networks.
+* *Full control*: You can deploy a cluster on infrastructure that you prepare and maintain, which provides maximum customizability. You can deploy clusters in connected or disconnected environments.
 
-The clusters have the following characteristics:
+Each method deploys a cluster with the following characteristics:
 
-* Highly available infrastructure with no single points of failure is available by default.
-* Administrators maintain control over what updates are applied and when.
+* Highly available infrastructure with no single points of failure, which is available by default.
+* Administrators can control what updates are applied and when.
 
 [id="about-the-installation-program"]
 == About the installation program
 
-You can use the installation program to deploy each type of cluster. The installation program generates main assets such as Ignition config files for the bootstrap, control plane (master), and worker machines. You can start an {product-title} cluster with these three configurations and correctly configured infrastructure.
+You can use the installation program to deploy each type of cluster. The installation program generates the main assets, such as Ignition config files for the bootstrap, control plane (master), and worker machines. You can start an {product-title} cluster with these three machine configurations, provided you correctly configured the infrastructure.
```

Comment:
```suggestion
You can use the installation program to deploy each type of cluster. The installation program generates the main assets, such as Ignition config files for the bootstrap, control plane, and compute machines. You can start an {product-title} cluster with these three machine configurations, provided you correctly configured the infrastructure.
```
Removing master/worker per conscious language guidelines.

Reply:
Thanks. Updated. 

I was unsure on this one as I still see "master" and "worker" dotted around in the docs. I know changing codebase names is out of our control, but general references are. Perhaps, I could update the complete install docs to replace the terms. WDYT?

Reply:
If you've got a ton of spare time, feel free! it would be one of those huge PRs where you post it in team-docs and a dozen people look over it